README.md

```diff
@@ -60,6 +60,12 @@ Please see https://stripe.com/docs/api for up-to-date documentation.
 
 ## Legacy Version Support
 
+### PHP 5.3
+
+If you are using PHP 5.3, you can download v5.8.0 ([zip](https://github.com/stripe/stripe-php/archive/v5.8.0.zip), [tar.gz](https://github.com/stripe/stripe-php/archive/v5.8.0.tar.gz)) from our [releases page](https://github.com/stripe/stripe-php/releases). This version will continue to work with new versions of the Stripe API for all common uses.
```

Comment:
Is it worth adding a scary blurb about how bad using this version (and 5.2) is?

Reply:
Couldn't hurt!